File: blog_springboot/src/main/java/com/nap_blog/entity/Blocked.java
package com.nap_blog.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Blocked {
    @TableId(type = IdType.AUTO)
    private Long id;
    private String ip;
    private String blockReason;
    private Integer status;
    private Date createDate;
}

File: blog_admin/types/Blocked.ts
import { type PageQuery } from '~/types/PageQuery'

export interface Blocked {
    id?: number
    ip: string
    blockReason:string
    status?:number
    createDate?: string
   
}
export interface BlockedQuery extends PageQuery {
    //关键字
    keyword?: string
    createDate?:string
}